src/app/ApplicationTemplate.Access/ApiClients/KillSwitch/IKillSwitchRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace ApplicationTemplate.DataAccess;
+
+/// <summary>
+/// Interface for the repository that gets when the kill switch is activated.
+/// </summary>
+public interface IKillSwitchRepository
+{
+	/// <summary>
+	/// Gets the status of the kill switch.
+	/// </summary>
+	void CheckKillSwitchActivated();
```

Comment:
Shouldn't this return a bool then?

Reply:
Also this should probably simply be `IsKillSwitchActive`

Reply:
And even then, I'm not sure you need that member, the observable is probably enough.

Reply:
It's probably good to have a get and observes, since we won't be able to observes firebase for now, unless we trigger the observable to check every x seconds?

Reply:
We can change it when we need to.
However even just an IObservable in the signature is likely enough, even if we need to do a _manual_ polling on some implementations.